src/main/java/uk/gov/justice/hmpps/prison/web/config/ConnectionProxyAopConfiguration.java

```diff
@@ -34,14 +32,6 @@ public class ConnectionProxyAopConfiguration {
     @Value("${oracle.tag.role.name}")
     private String tagUser;
 
-    @Bean
-    public OracleConnectionAspect oracleProxyConnectionAspect(
```

Comment:
Replaced with an annotation on the class as it doesn't need to know about defaultSchema anymore.